docs/transitions.md

```diff
@@ -0,0 +1,2 @@
+# Transitions
+The transition change state of object from one to another - that's helps in product lifecycle manegment. For each object (asset, support, licence) you can define some workflow (set of transitions) and provide special actions for each transitions.
```

Comment:
And decide if it's for developer or the user. I would think about the developer first, user will get user guide instead of focusing on transitions alone.